core/src/kvs/ds.rs

```diff
@@ -87,6 +89,9 @@ pub struct Datastore {
 	clock: Arc<SizedClock>,
 	// The index store cache
 	index_stores: IndexStores,
+	#[cfg(feature = "jwks")]
```

Comment:
I looked into this as we discussed, since I agree your point. However, since the property contains the `JwksCache` type, it needs to be conditional to the `jwks` feature being present, which introduces the `jwks` module. There are obviously alternatives, but I think this should not be an issue in the forseable future, and it is likely that the `jwks` feature will be absorbed into the default soon.